### ISSUE TITLE
Notebook tests can fail

### ISSUE THREAD

**Issue by opcode81** [jambit/sensAI#57]
See https://github.com/jambit/sensAI/runs/2884663556?check_suite_focus=true

@MischaPanch, are you familiar with such errors? Could this be related to the newly introduced caching (#31)?

**Comment by MischaPanch** [comment]
I don't think it has anything to do with caching. Seems to be related to this long standing issue:
https://github.com/jupyter/nbconvert/issues/1066

The developer promised to fix it some time soon, maybe we can just wait and restart the failing pipelines when this happens for the moment? Doesn't seem to happen all too often

**Comment by schroedk** [comment]
I got the same error while running two pipelines in parallel. While the first build was sucessful, the second one failed with the error message above. Rerunning the second build (after the first one finished) was sucessful. I will try to reproduce this behavior.

**Comment by opcode81** [comment]
@MischaPanch, [new failure encountered](https://github.com/jambit/sensAI/runs/3219457853?check_suite_focus=true#step:9:361).
Looks like the version of pytorch-lightning being used is incompatible with the version of torch being used; probably due to the relaxed version specification (`pytorch-lightning~=1.1`) in tox.ini.
